dynet/io.cc

```diff
@@ -84,37 +86,30 @@ void TextFileSaver::save(const LookupParameter & param,
 
 void TextFileSaver::save(const ParameterStorage & p,
                          const string & key) {
-  std::ostringstream buffer;
-  buffer.precision(FLOAT32_PRECISION);
-  buffer << dynet::as_vector(p.values) << endl;
+  datastream << "#Parameter# " << (key.size() > 0 ? key : p.name) << ' ' << p.dim << ' ';
+  size_t strsize = p.dim.size() * (FLOAT32_PRECISION + 8) + 1;
```

Comment:
If I understand correct, 8 here is for `+x.e-xx`. I think the value range is fine if there is no the spefical case.